app/lib/shared/Core/entities/DocumentEntity.js

```diff
@@ -483,6 +486,115 @@ export default class DocumentEntity {
         })
       })
     }
+    if (
+      hasTrackingRevisionHistory(doc) &&
+      hasTrackingVersionField(doc) &&
+      doc.document.tracking.revision_history.length > 0
+    ) {
+      if (
+        doc.document.tracking.revision_history
+          .slice()
+          .sort(
+            (a, z) => new Date(z.date).getTime() - new Date(a.date).getTime()
+          )[0].number !== doc.document.tracking.version
```

Comment:
According to [CSAF 6.1.6](https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/prose/csaf-v2-editor-draft.md#6116-latest-document-version) build metadata is ignored in the comparison.